packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -33,8 +33,8 @@ import { TimeSync } from '@iot-app-kit/react-components';
 import { debounce } from 'lodash';
 
 export type DashboardProperties = {
-  onSave: DashboardSave;
   onDashboardConfigurationChange?: DashboardConfigurationChange;
+  onSave?: DashboardSave;
```

Comment:
do we need to add a noop implementation in the header if they are using the default one?